src/nb_clean/cli.py

```diff
@@ -145,6 +145,7 @@ def clean(args: argparse.Namespace) -> None:
         notebook = nb_clean.clean_notebook(
             notebook,
             remove_empty_cells=args.remove_empty_cells,
+            remove_notebook_metadata=args.remove_notebook_metadata,
```

Comment:
I guess you're supposed to do the same for `check()` and `add_filter()`, and update the docstrings of all three functions.